src/Symfony/Component/Notifier/Exception/UnsupportedSchemeException.php

```diff
@@ -132,6 +132,10 @@ class UnsupportedSchemeException extends LogicException
             'class' => Bridge\MicrosoftTeams\MicrosoftTeamsTransportFactory::class,
             'package' => 'symfony/microsoft-teams-notifier',
         ],
+        'smsbiuras' => [
+            'class' => Bridge\SmsBiuras\SmsBiurasTransportFactory::class,
+            'package' => 'symfony/smsbiuras-notifier',
```

Comment:
```suggestion
            'package' => 'symfony/sms-biuras-notifier',
```

please rename the package name like this, thanks